### DISCUSSION TITLE
Retrospective: I finished review 2021 academic conference chapters each books. (special thanks to @Joseph-Cha)

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#169] [General]
## Origin objectives.

작년 6월 마지막 academic conference 2021을 마치고 저는 세 분에게 나머지 책을 읽고 리뷰를 써 보는 건 어떤지에 대한 의견을 드렸습니다.

- #54 

하지만 의견을 주신 분이 없어서 저 혼자 진행하기로 했고 @Joseph-Cha 님은 그 당시 제 옆자리에 계신 분이어서 오프라인으로 reviewer는 해줄 수 있다고 하셔서 그렇게 호기롭게 진행하게 되었습니다.

## Delay, delay, delay... give up or despair

최초 계획은 8월, 9월, 10월 해서 한 달에 한 권 씩 끝내는 일정으로 잡았지만 일정은 지속적으로 밀렸습니다. 
일정이 밀려서 밀린 흑역사의 기록은 milestone에 그대로 드러나 있습니다. ㅎㅎ

- https://github.com/ThinkAboutSoftware/AcademicConference/milestone/2

<img width="421" alt="image" src="https://user-images.githubusercontent.com/17442457/155934025-36c9505e-ceaa-413f-8a9d-ef6ecd415a9d.png">

## Good things

무엇보다도 하기로 한 리뷰를 모두 마쳤다는 점입니다.

이것을 해냈다는 걸로 저는 제 스스로에게 칭찬을 해주고 싶습니다.

사실 돌이켜 보면 제대로 된 milestone을 계획하고 issue별 chapter를 따로 두어 작은 실천 분량으로 해 보면 정말 된다는 생각을 하게 해주기도 한 것 같습니다.

그리고 동료에게도 작은 업무 단위로 나눠서 하라는 얘기를 조금 더 경험적으로 얘기해 줄 수 있지 않을까 하는 생각도 해봅니다.

## Future improvement

계획을 세워도 계획대로 진행하지 못한 부분을 반성해 봤습니다.
확실히 일정이 바쁜 것도 있었고, 제가 게으른 탓도 있었고 여러가지 복합적인 문제가 있었던 것 같습니다.
그렇다고 하더라도 계획은 세워 둬야 나중에라도 해야 한다는 의지를 발휘할 수 있는 것 같습니다.
그리고 내가 리뷰해야 할 내용이 얼마나 남아 있고 언제 까지 할 수 있는지에 대한 계획은 세워두는게 확실히 좋다 라는 걸 다시 한번 느꼈습니다.

## At last

제 2022년 새해 목표는 아래와 같습니다.

`지금 당장 해야 하는 일을 하자`

입니다.

그게 개발이 되었던 책 리뷰가 되었든 사람들에게 해야 하는 이야기이든지 말이죠.

이렇게 흑역사(?)가 언젠가는 청산이 가능하다는 점에서 제가 2020년, 그리고 2021년에 벌여 놓은 흑역사들을 이제 느리지만 꾸준하게 청산해 보려 합니다..

몇 달 혹은 몇 년이 걸릴지 모를 일이긴 합니다.
하지만 지속적으로 조금씩 뭔가 해 나간다면 달성률은 채워질 것이며
흑역사는 청산 되리라는게 제 믿음입니다.

마지막으로 @Joseph-Cha 님에게 이틀 전 까지의 academic conference 2021의 모든 pull request에 reviewer로써 충실히 리뷰 의견을 달아 주셨고, 마지막 까지 리뷰 달아 주셔서 감사의 말씀을 다시 드립니다.

**Comment by Joseph-Cha** [comment]
무한 축하를 날려드립니다👏👏👏

리뷰 하는 재미도 나름 쏠쏠했어요.
올해 아카데미 컨퍼런스도 끝까지 잘 마무리 할 수 있으면 좋겠습니다!!
